L1Trigger/L1TCaloLayer1/src/UCTTower.hh

```diff
@@ -5,24 +5,26 @@
 
 #include <vector>
 
-#define etInputMax 0xFF
-
-#define etMask 0x000001FF
-#define erMask 0x00000E00
-#define erMaxV 7
-#define erShift 9
-#define zeroFlagMask 0x00001000
-#define eohrFlagMask 0x00002000
-#define hcalFlagMask 0x00004000
-#define ecalFlagMask 0x00008000
-#define stg2BitsMask 0x0000FFFF
-#define miscBitsMask 0x0000F000
-#define miscShift 12
-
-#define ecalBitsMask 0x00FF0000
-#define ecalShift 16
-#define hcalBitsMask 0xFF000000
-#define hcalShift 24
+namespace {
```

Comment:
I'm not sure using an anonymous namespace is the right choice,  but maybe @davidlange6 can comment.

My impression is that it does protect from linker clashes (the symbols should visible only within the file that defines them), but not from clashes with other symbols with the same name.